proto/greptime/v1/database.proto

```diff
@@ -54,6 +54,10 @@ message QueryRequest {
   }
 }
 
+message InsertRequests {
+  repeated InsertRequest inserts = 1;
+}
+
```

Comment:
It seems we also need InsertResponses:
```
message InsertResponses {
  repeated AffectedRows affected_rows = 1;
}
```

Reply:
No, that's the "nodelay" optimization in the future.